CHANGELOG.md

```diff
@@ -51,6 +51,8 @@ When releasing, please add the new-release-boilerplate to docs/pallas/CHANGELOG.
   * The internal utilities `jax.core.check_eqn`, `jax.core.check_type`, and
     `jax.core.check_valid_jaxtype` are now deprecated, and will be removed in
     the future.
+  * Deprecated calling {func}`jax.dlpack.from_dlpack` with a DLPack tensor. It
```

Comment:
Rephrase a bit better:
* Passing a DLPack capsule to {func}`jax.dlpack.from_dlpack` is deprecated. The argument to `from_dlpack` should be an array from another framework that implements the `__dlpack__` protocol.

Reply:
Same in the deprecation warning.

Reply:
Done.